### DISCUSSION TITLE
Needs and the Needs universe

### DISCUSSION THREAD

**Discussion by kreuzberger** [useblocks/sphinx-needs#1356] [General]
Staring with sphinx-needs for my requirements, the whole need/tracking workflow required to use many of the useblocks provided repositories.
These are in my case sphinx-simplepdf, sphinx-test-reports, sphinx-collections etc. I also use libpdf for testing my documents.

Currently there seems to be no "main" repository or "main" responsiblity to check that all the useblocks provide sphinx extensions

* work and build against the same dependencies
* work and build against the "released" versions.

See e.g.  https://github.com/useblocks/sphinx-test-reports/issues/78 or
https://github.com/useblocks/sphinx-needs/issues/1331

There should be a overall check for the released versions of the useblocks provided modules to provide an overall consistency.

**Comment by kreuzberger** [comment]
This also e.g. affects the excluding requirements for sphinx-needs and libpdf regarding their dependencies

**Comment by danwos** [comment]
Totally agree, we also have identified this as a problem.
And we are trying to find some time to fix this. We are on it.
